cocoa/src/toga_cocoa/app.py

```diff
@@ -385,3 +385,6 @@ def get_current_window(self):
 
     def set_current_window(self, window):
         window._impl.native.makeKeyAndOrderFront(window._impl.native)
+
+    def focus(self):
+        self.native.activateIgnoringOtherApps_(True)
```

Comment:
The trailing underscore isn't needed here; it's a valid but historical syntax. 
```suggestion
        self.native.activateIgnoringOtherApps(True)
```